app/components/VolunteerApproval/RequestingHours.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Button, ButtonToolbar, Input, OverlayTrigger, Popover} from 'react-bootstrap';
+
+var HourPopover =
+    <Popover id="hrconfirm" title="Hour Confirmation">
+        <div>
+            <h5>Start Time:</h5>
+            <h5>End Time:</h5>
+            <h5>Total: </h5>
+            <Button> Submit </Button>
+        </div>
+    </Popover>
+
+
+class RequestingHours extends Component {
+
+    render() {
+        return (
+            <OverlayTrigger trigger='click' placement='bottom' overlay={HourPopover}>
+                <img src='http://placekitten.com/g/400/200' />
+            </OverlayTrigger>
+        );
+    }
+}
+
```

Comment:
No need to extend Component for render, can simply be a pure function, same is true for all the other components

```javascript
const  RequestingHours = () (
    <OverlayTrigger trigger='click' placement='bottom' overlay={HourPopover}>
         <img src='http://placekitten.com/g/400/200' />
     </OverlayTrigger>
)
```